### PR TITLE
New content

### PR DESCRIPTION
**Pull request by 0xricksanchez**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -122,13 +122,17 @@ s](http://www.s3.eurecom.fr/docs/usenixsec14_costin.pdf)
 - [Binary Hardening in IoT products](https://cyber-itl.org/2019/08/26/iot-data-writeup.html)
 - [Cracking Linksys “Encryption”](http://www.devttys0.com/2014/02/cracking-linksys-crypto/)
 - [Deadly Sins Of Development](https://youtu.be/nXyglaY9N9w) - Conference talk presenting several real world examples on real bad implementations :tv:.
+- [Dumping firmware a device with a buspirate and a SPI flash chip](https://www.iotpentest.com/2019/06/dumping-firmware-from-device-using.html)
```
Comment (weidenba) [inline readme.md:125]: link is not working and I think there is a word or something missing 

> Dumping firmware a device ... 

sounds wired.

File: readme.md [hunk 2]
```diff
@@ -122,13 +122,17 @@ s](http://www.s3.eurecom.fr/docs/usenixsec14_costin.pdf)
...
 - [Hacking the DSP-W215, Again](http://www.devttys0.com/2014/05/hacking-the-dspw215-again/)
 - [Hacking the PS4](https://cturt.github.io/ps4.html) - Introduction to PS4's security.
+- [JTAG Explained](https://blog.senr.io/blog/jtag-explained#) - A walkthrough covering UART and JTAG bypassing a protected login shell.
 - [Multiple vulnerabilities found in the D-link DWR-932B](https://pierrekim.github.io/blog/2016-09-28-dlink-dwr-932b-lte-routers-vulnerabilities.html)
 - [Pwning the Dlink 850L routers and abusing the MyDlink Cloud protocol](https://pierrekim.github.io/blog/2017-09-08-dlink-850l-mydlink-cloud-0days-vulnerabilities.html)
 - [PWN Xerox Printers (...again)](https://www.fkie.fraunhofer.de/content/dam/fkie/de/documents/xerox_phaser_6700_white_paper.pdf)
+- [Reverse Engineering Serial Ports](http://www.devttys0.com/2012/11/reverse-engineering-serial-ports/)
 - [Reversing Firmware With Radare](https://www.bored-nerds.com/reversing/radare/automotive/2019/07/07/reversing-firmware-with-radare.html)
 - [Reversing the Huawei HG533](http://jcjc-dev.com/2016/04/08/reversing-huawei-router-1-find-uart/)
+- [Samsung Firmware Magic](https://github.com/chrivers/samsung-firmware-magic)
```
Comment (weidenba) [inline readme.md:135]: I think this is tool and not a case study. If it was a case study I would expect a description of the reversing process.

File: readme.md [hunk 3]
```diff
@@ -150,11 +154,14 @@ s](http://www.s3.eurecom.fr/docs/usenixsec14_costin.pdf)
 
 ### Blogs
 <!--lint ignore no-repeat-punctuation-->
+- [Azeria Lab](https://azeria-labs.com/writing-arm-assembly-part-1/)
```
Comment (weidenba) [inline readme.md:157]: This should be part of the tutorials section as well

File: readme.md [hunk 4]
```diff
@@ -122,13 +122,17 @@ s](http://www.s3.eurecom.fr/docs/usenixsec14_costin.pdf)
 - [Binary Hardening in IoT products](https://cyber-itl.org/2019/08/26/iot-data-writeup.html)
 - [Cracking Linksys “Encryption”](http://www.devttys0.com/2014/02/cracking-linksys-crypto/)
 - [Deadly Sins Of Development](https://youtu.be/nXyglaY9N9w) - Conference talk presenting several real world examples on real bad implementations :tv:.
+- [Dumping firmware a device with a buspirate and a SPI flash chip](https://www.iotpentest.com/2019/06/dumping-firmware-from-device-using.html)
 - [Hacking the DSP-W215, Again](http://www.devttys0.com/2014/05/hacking-the-dspw215-again/)
 - [Hacking the PS4](https://cturt.github.io/ps4.html) - Introduction to PS4's security.
+- [JTAG Explained](https://blog.senr.io/blog/jtag-explained#) - A walkthrough covering UART and JTAG bypassing a protected login shell.
```
Comment (weidenba) [inline readme.md:128]: We should add a new category "Tutorials". This should be part of it.

File: readme.md [hunk 5]
```diff
@@ -122,13 +122,17 @@ s](http://www.s3.eurecom.fr/docs/usenixsec14_costin.pdf)
...
 - [Multiple vulnerabilities found in the D-link DWR-932B](https://pierrekim.github.io/blog/2016-09-28-dlink-dwr-932b-lte-routers-vulnerabilities.html)
 - [Pwning the Dlink 850L routers and abusing the MyDlink Cloud protocol](https://pierrekim.github.io/blog/2017-09-08-dlink-850l-mydlink-cloud-0days-vulnerabilities.html)
 - [PWN Xerox Printers (...again)](https://www.fkie.fraunhofer.de/content/dam/fkie/de/documents/xerox_phaser_6700_white_paper.pdf)
+- [Reverse Engineering Serial Ports](http://www.devttys0.com/2012/11/reverse-engineering-serial-ports/)
```
Comment (weidenba) [inline readme.md:132]: This should go to the "Tutorials" section as well

File: readme.md [hunk 6]
```diff
@@ -150,11 +154,14 @@ s](http://www.s3.eurecom.fr/docs/usenixsec14_costin.pdf)
 
 ### Blogs
 <!--lint ignore no-repeat-punctuation-->
+- [Azeria Lab](https://azeria-labs.com/writing-arm-assembly-part-1/)
 - [/dev/ttyS0's Embedded Device Hacking](http://www.devttys0.com/blog/)
 - [Exploiteers](https://www.exploitee.rs/)
 - [Hackaday](https://hackaday.com)
 - [jcjc's Hack The World](https://jcjc-dev.com/)
+- [Mikroe - UART explained](https://www.mikroe.com/blog/uart-serial-communication)
```
Comment (weidenba) [inline readme.md:162]: Tutorial section?

File: readme.md (hunk 1 above)
Comment (0xricksanchez) [reply readme.md]: Fixed description, Link works for me tho

File: readme.md (hunk 2 above)
Comment (0xricksanchez) [reply readme.md]: Moved to a new "Misc Tools" Category for such tools

File: readme.md (hunk 4 above)
Comment (0xricksanchez) [reply readme.md]: done

File: readme.md (hunk 6 above)
Comment (0xricksanchez) [reply readme.md]: done

File: readme.md (hunk 5 above)
Comment (0xricksanchez) [reply readme.md]: done

File: readme.md (hunk 3 above)
Comment (0xricksanchez) [reply readme.md]: done

**Pull request merged**